web/src/copy_and_paste.js

```diff
@@ -293,29 +293,138 @@ function get_end_tr_from_endc($endc) {
     return $endc.parents(".selectable_row").first();
 }
 
+function cleanAttribute(attribute) {
+    return attribute ? attribute.replace(/(\n+\s*)+/g, "\n") : "";
+}
```

Comment:
This function would benefit from a comment explaining exactly what we're expecting it to do.